front-end/src/components/LikeListItem.tsx

```diff
@@ -0,0 +1,20 @@
+import { Card, CardBody } from "reactstrap";
+import { Like } from "../types/Like";
+
+interface Props {
+  like: Like;
+}
+
+/**
+ * Author list component to show list of authors
+ * @param props
+ */
+export default function LikelListItem(props: Props) {
```

Comment:
nit; Is the naming of this a typo? `LikelListItem` -> `LikeListItem`?

Reply:
Good catch :+1: